.pipelines/templates/e2e-test.yaml

```diff
@@ -72,3 +72,5 @@ steps:
         CI_KIND_CLUSTER: ${{ parameters.ciKindCluster }}
       ${{ if parameters.isArcTest }}:
         IS_ARC_TEST: ${{ parameters.isArcTest }}
+      ${{ if parameters.testReleasedVersion }}:
+        GINKGO_SKIP: WriteCertAndKeyInSeparateFiles
```

Comment:
Why are we skipping this?

Reply:
If the image is a released versions (i.e <= `v1.3`), it still doesn't support the split cert/key feature, so we need to skip tests for those versions.

Reply:
added this as a comment in the file